test/e2e/changedFilesUtil.js

```diff
@@ -20,7 +20,7 @@ async function readChangedFiles() {
     return changedFiles;
   } catch (error) {
     console.error('Error reading from file:', error);
-    return '';
+    return [];
```

Comment:
returning an empty array, in order to handle properly the subsequent filter, when there is no file (in develop, master and RC branch)